### PR TITLE
Add a pull request template

### PR DESCRIPTION
**Pull request by SamDepardieu**

### CODE REVIEW

**Comment by SamDepardieu** [comment]
[Create a PR template for the parser](https://trello.com/c/N4McdFoQ/369-create-a-pr-template-for-the-parser)

File: pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Description
+
+Please include a summary of the changes this PR introduces for the codebase. Make sure to specify is some sections needs special attention,
```
Comment (jdkram) [inline pull_request_template.md]: > Make sure to specify if some sections need special attention, and why you want to introduce this change.

File: pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+as well as why you want to introduce this change.
+
+Make sure to split changes across multiple pull request, as we won review meta-pull requests.
```
Comment (jdkram) [inline pull_request_template.md]: > Make sure to split changes across multiple pull requests, as we won't review meta-pull requests.

Is "meta" an accessible enough word? Maybe "bundled pull requests."

File: pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+Finally, make sure your PR follows our code of conduct before posting (Check our [contributing guidelines](CONTRIBUTING.md) if you're not sure).
+
+Fixes # (issue)
```
Comment (jdkram) [inline pull_request_template.md]: Should `Fixes # (issue)` go here, or would it be a more natural fit in the _Type of change_ section?

File: pull_request_template.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+## Type of change
+
+Please delete options that are not relevant.
+
+- [ ] :bug: Bug fix
+- [ ] :sparkles: New feature
+- [ ] :fire: Breaking change
+- [ ] :memo: Documentation update
+
```
Comment (jdkram) [inline pull_request_template.md]: What should we advise them to do if they want to submit a PR that doesn't match any of these?

File: pull_request_template.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+# How Has This Been Tested?
+
+Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration:
+
+**Test Configuration**:
+* Firmware version:
+* Hardware:
+* Toolchain:
```
Comment (jdkram) [inline pull_request_template.md]: I'm unsure what Toolchain means here. Is it worth having italicised sample text for each of these?

File: pull_request_template.md (hunk 4 above)
Comment (SamDepardieu) [reply pull_request_template.md]: Changing it to more relevant items 👍

File: pull_request_template.md [hunk 6]
```diff
@@ -1,19 +1,17 @@
 # Description
 
-Please include a summary of the changes this PR introduces for the codebase. Make sure to specify is some sections needs special attention,
-as well as why you want to introduce this change.
+Please include a summary of the changes this PR introduces for the codebase. Make sure to specify if some sections need special attention
+, and why you want to introduce this change.
 
-Make sure to split changes across multiple pull request, as we won review meta-pull requests.
+Make sure to split changes across multiple pull request, as we won review  bundled pull requests.
```
Comment (jdkram) [inline pull_request_template.md]: *won't

**Comment by nsorros** [comment]
The template is lacking instructions on how to reference the issue they are solving, if this is relevant.

**Comment by lizgzil** [comment]
This LGTM, but I am a bit unsure on something - do the people copy and paste and edit this template into the description of the PR then? I assume so, but maybe you should clarify that this is what's expected at the beginning?

**Comment by SamDepardieu** [comment]
@lizgzil If you're unsure of what to do with this, then it's worth specifying it 😄

File: pull_request_template.md [hunk 7]
```diff
@@ -0,0 +1,31 @@
+# Description
+
+Please include a summary of the changes this PR introduces for the codebase.
+Also specify if some sections need special attention, and why you want to introduce this change.
+
+Make sure to split changes across multiple pull request, as we won't review  bundled pull requests.
```
Comment (lizgzil) [inline pull_request_template.md]: 'requests', and double space between review and bundled

File: pull_request_template.md [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+Please delete options that are not relevant.
+
+- [ ] :bug: Bug fix (Add `Fix #(issue)` to your PR)
+- [ ] :sparkles: New feature
+- [ ] :fire: Breaking change
+- [ ] :memo: Documentation update
+
+# How Has This Been Tested?
+
+Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration:
```
Comment (lizgzil) [inline pull_request_template.md]: 'reproduce it'

**Comment by SamDepardieu** [comment]
@jdkram @nsorros @lizgzil Another round? (Hopefully the last one 😄 )

File: pull_request_template.md [hunk 9]
```diff
@@ -3,7 +3,7 @@
 Please include a summary of the changes this PR introduces for the codebase.
 Also specify if some sections need special attention, and why you want to introduce this change.
 
-Make sure to split changes across multiple pull request, as we won't review  bundled pull requests.
+Make sure to split changes across multiple pull request, as we won't review bundled pull requests.
```
Comment (jdkram) [inline pull_request_template.md]: plural: 
Make sure to split changes across multiple pull request**s**,

File: pull_request_template.md [hunk 10]
```diff
@@ -18,7 +18,7 @@ Please delete options that are not relevant.
 
 # How Has This Been Tested?
 
-Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration:
+Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce it. Please also list any relevant details for your test configuration:
```
Comment (jdkram) [inline pull_request_template.md]: "it" implies singular, might be clearer as:
"Provide instructions so we can run the tests."
or "Provide instructions so we can reproduce your tests."

**Comment by SamDepardieu** [comment]
@jdkram Fixed

**Comment by jdkram** [comment]
@SamDepardieu 🎉

**Comment by SamDepardieu** [comment]
Still need two other approvals 😢

**Comment by jdkram** [comment]
@SamDepardieu see if they leave their laptops unlocked in Datalabs...

**Pull request merged**